CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 0.22.0
+
+*July 16th, 2018*
+
+BREAKING CHANGES
+* [x/gov] Increase VotingPeriod, DepositPeriod, and MinDeposit
+
+IMPROVEMENTS
+* [gaiad] Default config updates:
+    - `timeout_commit=5000` so blocks only made every 5s
+    - `prof_listen_addr=localhost:6060` so profile server is on by default
+    - `p2p.send_rate` and `p2p.recv_rate` increases 10x (~5MB/s)
```

Comment:
Have these changed? The before/after config values are both `5120000`. Or did we miss the changelog last time?

Reply:
Changelog entry is in v0.20 but I don't think it actually took effect, so it was just using the Tendermint defaults